### ISSUE TITLE
Git Bash support on Windows ?

### ISSUE THREAD

**Issue by nkzawa** [zeit/now#29]
See: https://github.com/zeit/now/issues/25

**Comment by e-beach** [comment]
Although `setRawMode` is necessary to detect input before the user hits the enter key, `now` shouldn't require this feature. For example, the email-prompter should read in an email but offer now prompting in this case.
Environments without `setRawMode` include Git-Bash, IDEs (at least anything JetBrains), and cloud computing like `repl.it`, which seems an awful lot to lose for tab completion.

**Comment by matheuss** [comment]
@e-beach yeah it's very easy fix – I'll implement it today 😄 Just a matter of accepting whatever needed on `argv`, and printing a nice message if `setRawMode` is not available 👌
